### PR TITLE
fixed lint errors

### PR DESCRIPTION
**Pull request by nehajain008**
Story [AB#14286](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14286)
Task [AB#14360](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14360)

### CODE REVIEW

File: app/pages/complete-order/contextCreator.test.js [hunk 1]
```diff
@@ -3,19 +3,20 @@ import withoutFundingManifest from './withoutFundingManifest.json';
...
+      const context = getContext({ orderId: 'order-1', fundingSource: true, odsCode: someOdsCode });
       expect(context.backLinkText).toEqual(withFundingManifest.backLinkText);
     });
 
     it('should construct the backLinkHref', () => {
       const orderId = 'order-id';
-      const context = getContext({ orderId, fundingSource });
-      expect(context.backLinkHref).toEqual(`${baseUrl}/organisation/${orderId}`);
+      const context = getContext({ orderId, fundingSource, odsCode: someOdsCode });
+      expect(context.backLinkHref).toEqual(`${baseUrl}/organisation/${someOdsCode}/${orderId}`);
```
Comment (kiran-mastek) [inline app/pages/complete-order/contextCreator.test.js:19]: This doesn't seem correct - as per the story the desired URL should have the work 'order' between `odsCode` and `orderId`:
```Existing Order URL = ".../order/organisation/C010000-01"```
```Desired Order URL Example = ".../order/organisation/13Y/order/C010000-01"```

Comment (nehajain008) [reply app/pages/complete-order/contextCreator.test.js]: Updated urls with order before orderid

File: app/pages/sections/order-items/additional-services/dashboard/contextCreator.test.js [hunk 2]
```diff
@@ -7,21 +7,22 @@ import { baseUrl } from '../../../../../config';
...
     it.each`
     senderUrl                               |  expectedUrl                                                | selectedPrice                   | catalogueItemExists
     ${`${somefakeUrl}/items/894`}           | ${additionalServicesUrl}                                    | ${mockSelectedPricePatientType} | ${''}
     ${''}                                   | ${additionalServicesUrl}                                    | ${mockSelectedPricePatientType} | ${''}
     ${`${somefakeUrl}/date`}                | ${`${somefakeUrl}/date`}                                    | ${mockSelectedPricePatientType} | ${''}
     ${`${somefakeUrl}/additional-services`} | ${`${somefakeUrl}/additional-services`}                     | ${mockSelectedPricePatientType} | ${''}
-    ${`${somefakeUrl}/recipients`}          | ${`${baseUrl}/organisation/${orderId}/additional-services`} | ${mockSelectedPricePatientType} | ${''}
+    ${`${somefakeUrl}/recipients`}          | ${`${baseUrl}/organisation/${odsCode}/order/${orderId}/additional-services`} | ${mockSelectedPricePatientType} | ${''}
```
Comment (bjss-chris) [inline app/pages/sections/order-items/additional-services/dashboard/contextCreator.test.js:25]: Can you fix the alignment of the vertical bars in the table?

File: app/pages/sections/order-items/associated-services/order-item/contextCreator.test.js [hunk 3]
```diff
@@ -9,20 +9,23 @@ describe('associated-services order-item contextCreator', () => {
...
     const mockAssociatedServicePrices = { prices: [{ priceId: 29 }, { priceId: 30 }] };
     const somefakeUrl = 'https://some.url.co.uk/order-id';
+    const odsCode = '03F';
     it.each`
     senderUrl                               |  expectedUrl                           | associatedServicePrices
     ${`${somefakeUrl}/associated-service`}  | ${`${somefakeUrl}/associated-service`} | ${''}   
     ${`${somefakeUrl}/price`}               | ${`${somefakeUrl}/price`}              | ${''}  
     ${`${somefakeUrl}/associated-services`} | ${`${somefakeUrl}/associated-services`}| ${''}   
-    ${`${somefakeUrl}/neworderitem`}        | ${`${baseUrl}/organisation/${orderId}/associated-services/select/associated-service/price`} |${mockAssociatedServicePrices}   
+    ${`${somefakeUrl}/neworderitem`}        | ${`${baseUrl}/organisation/${odsCode}/order/${orderId}/associated-services/select/associated-service/price`} |${mockAssociatedServicePrices}
```
Comment (bjss-chris) [inline app/pages/sections/order-items/associated-services/order-item/contextCreator.test.js:18]: Can you fix the alignment of the vertical bars in the table?

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Good work. There's just a couple of unit test tables where the vertical bar alignment needs fixing.

**Comment by bjss-chris** [comment]
@nehajain008, can we close this PR now?

**Comment by nehajain008** [comment]
> 
> 
> @nehajain008, can we close this PR now that we've merged #645?

Yes @bjss-chris , closing this PR.

**Pull request closed without merging**